client/js/widgets/button.js

```diff
@@ -382,6 +384,8 @@ export class Button extends Widget {
               return w.p(a.property) >= a.value;
             else if(a.relation === '>')
               return w.p(a.property) > a.value;
+            else if(a.relation === 'in' && Array.isArray(a.value))
+              return a.value.indexOf(w.p(a.property)) != -1;
```

Comment:
If someone writes `"in": "<holder name>"`, for example, instead of `"in": ["<holder name>"]`, this probably won't work as expected. I realize this is a contrived example, but what *should* the behavior be if `in` is not fed an array for its value? Right now, the relation appears to be silently ignored.

Reply:
Did you test that and it was ignored? Because the code should fall back to `"relation": "=="` and show a warning `Warning: Relation in interpreted as ==.` in the button debug output.

Sure, that message could be a bit clearer but I think that's reasonable behaviour..

Reply:
No, all I did was (mis) read the code. I agree, it should fall back to ==, and that seems like exactly what would be expected.